src/sentry/api/authentication.py

```diff
@@ -2,6 +2,7 @@
 
 from django.contrib.auth.models import AnonymousUser
 from django.utils.crypto import constant_time_compare
+from django.utils.encoding import force_text
```

Comment:
This is deprecated as of Django 3.0: https://docs.djangoproject.com/en/3.1/ref/utils/#django.utils.encoding.force_text

Prefer `force_str` which is also available in 1.11?
```suggestion
from django.utils.encoding import force_str
```

Reply:
IMO let's migrate this over all at once, instead of having a mix for now.